### ISSUE TITLE
Error Prone static analysis plugin error on java 11.0.9

### ISSUE THREAD

**Issue by niteshchaudhary** [google/error-prone#1955]
### Description of the problem / feature request:

> Error Prone static analysis plugin error on java 11.0.9

### Feature requests: what underlying problem are you trying to solve with this feature?

> I want to use Java on my mac with dual mode (java 8 and java 11)

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Follow below steps
> 1. brew tap AdoptOpenJDK/openjdk
>2. brew cask install homebrew/cask-versions/adoptopenjdk8
>3. brew cask install adoptopenjdk11
>4. Add below in bash-profile:
```
export JAVA_8_HOME=$(/usr/libexec/java_home -v 1.8)
export JAVA_11_HOME=$(/usr/libexec/java_home -v 11)

# Easy command for setting the JDK version to 8
alias java8='export JAVA_HOME=$JAVA_8_HOME'
# Easy command for setting the JDK version to 11
alias java11='export JAVA_HOME=$JAVA_11_HOME'

# Set the default version here
# export JAVA_HOME=$JAVA_8_HOME
export JAVA_HOME=$JAVA_11_HOME
```
> 5. When I try to build my app (mvn clean install), I get the below error.

```
error: An unhandled exception was thrown by the Error Prone static analysis plugin.
[ERROR]      Please report this at https://github.com/google/error-prone/issues/new and include the following:
[ERROR]
[ERROR]      error-prone version: 2.3.2-HA-p1
[ERROR]      BugPattern: TypeParameterUnusedInFormals
[ERROR]      Stack Trace:
[ERROR]      java.lang.NoSuchFieldError: bound
[ERROR]   	at com.google.errorprone.bugpatterns.TypeParameterUnusedInFormals.matchMethod(TypeParameterUnusedInFormals.java:73)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScanner.visitMethod(ErrorProneScanner.java:890)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScanner.visitMethod(ErrorProneScanner.java:146)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCMethodDecl.accept(JCTree.java:898)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreePathScanner.scan(TreePathScanner.java:82)
[ERROR]   	at com.google.errorprone.scanner.Scanner.scan(Scanner.java:83)
[ERROR]   	at com.google.errorprone.scanner.Scanner.scan(Scanner.java:43)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.scanAndReduce(TreeScanner.java:90)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.scan(TreeScanner.java:105)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.scanAndReduce(TreeScanner.java:113)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.visitClass(TreeScanner.java:187)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScanner.visitClass(ErrorProneScanner.java:588)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScanner.visitClass(ErrorProneScanner.java:146)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCClassDecl.accept(JCTree.java:808)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreePathScanner.scan(TreePathScanner.java:82)
[ERROR]   	at com.google.errorprone.scanner.Scanner.scan(Scanner.java:83)
[ERROR]   	at com.google.errorprone.scanner.Scanner.scan(Scanner.java:43)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.scan(TreeScanner.java:105)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.scanAndReduce(TreeScanner.java:113)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreeScanner.visitCompilationUnit(TreeScanner.java:144)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScanner.visitCompilationUnit(ErrorProneScanner.java:603)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScanner.visitCompilationUnit(ErrorProneScanner.java:146)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCCompilationUnit.accept(JCTree.java:591)
[ERROR]   	at jdk.compiler/com.sun.source.util.TreePathScanner.scan(TreePathScanner.java:56)
[ERROR]   	at com.google.errorprone.scanner.Scanner.scan(Scanner.java:65)
[ERROR]   	at com.google.errorprone.scanner.ErrorProneScannerTransformer.apply(ErrorProneScannerTransformer.java:43)
[ERROR]   	at com.google.errorprone.ErrorProneAnalyzer.finished(ErrorProneAnalyzer.java:151)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.api.MultiTaskListener.finished(MultiTaskListener.java:132)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1418)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1365)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:960)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:311)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:170)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.Main.compile(Main.java:57)
[ERROR]   	at jdk.compiler/com.sun.tools.javac.Main.main(Main.java:43)
```

### What version of Error Prone are you using?

> 2.3.2-HA-p1

###  Have you found anything relevant by searching the web?

> No

### when I run `java -version`.
> output:
java -version
openjdk version "11.0.9.1" 2020-11-04
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.9.1+1)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.9.1+1, mixed mode)

**Comment by cushon** [comment]
Duplicate of #1352, fixed in 2.3.4

**Comment by niteshchaudhary** [comment]
What is the solution? please advice what should I do?

**Comment by cushon** [comment]
You're using an old version of Error Prone based on `2.3.2`. (I'm not sure where the `-HA-p1` suffix is coming from, is this a custom build?)

The bug you reported is a duplicate of #1352, which is fixed in Error Prone version 2.3.4.

I recommend updating to the latest version of Error Prone.
